core/src/main/java/org/elasticsearch/search/fetch/subphase/highlight/HighlightBuilder.java

```diff
@@ -522,4 +525,33 @@ public String toString() {
             return name().toLowerCase(Locale.ROOT);
         }
     }
+
+    public static enum BoundaryScannerType implements Writeable {
+        SIMPLE, BREAK_ITERATOR;
```

Comment:
BREAK_ITERATOR is not reflecting what this type is doing. Maybe just SENTENCE ? Or SENTENCE_BREAK ?